backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/compatibility/properties/EEnumIfDescriptionProvider.java

```diff
@@ -83,13 +84,25 @@ private SelectDescription getSelectDescription() {
                 .labelProvider(this.getLabelProvider())
                 .valueProvider(this.getValueProvider())
                 .optionsProvider(this.getOptionsProvider())
+                .valueRequiredProvider(this.getValueRequiredProvider())
```

Comment:
Do we need to have such special use case, couldn't it be done using simply another value in the value provider and thus without hardcoding "null" in the frontend? I've always believed that this hardcoded "null" was odd and while this change may work, it may be a hack on top of a previous hack.

Reply:
I don't know. It seems to me that _unsetting_ a feature is fundamentally different from setting it to a value, even a "special" one. Even on the frontend it wouldn't shock me to have a clearly distinct way to indicate that a property is _not set_ and to _unset_ it (not that I'm advocating for it right now).

See for example text fields, where it is impossible to distinguish in the UI if the value is unset of set explicitly to `""`. And once someone has set a value to a text attribute once it is impossible in the UI to unset it (you can only set it to empty, but that is not always the same thing).

I know it's a different widget that this one, but it illustrates that IMO "no value" is something semantically very different from "a value which can be interpreted as 'empty'", and the UI should support disinguishing between the two.